Formula/gupnp-tools.rb

```diff
@@ -22,18 +23,16 @@ class GupnpTools < Formula
   depends_on "gtksourceview4"
   depends_on "gupnp"
   depends_on "gupnp-av"
-  depends_on "libsoup@2"
+  depends_on "libsoup"
```

Comment:
Can try to backport https://gitlab.gnome.org/GNOME/gupnp-tools/-/commit/7ee5fd9beb04cdb5ebf76b3bf79b132aaacc5e11